src/NHibernate/Async/Transaction/AdoNetTransactionFactory.cs

```diff
@@ -40,35 +40,25 @@ public virtual Task ExecuteWorkInIsolationAsync(ISessionImplementor session, IIs
 			async Task InternalExecuteWorkInIsolationAsync()
 			{
 
-				DbConnection connection = null;
 				DbTransaction trans = null;
-				// bool wasAutoCommit = false;
```

Comment:
I have added a small cleanup commit.
Afaik, the auto-commit feature and the transaction manager are not ported into NHibernate, so those decade old comments and todo-s are unlikely to be done any time soon. They are just polluting the code in my view.